cloudflare.tf

```diff
@@ -0,0 +1,45 @@
+
+data "cloudflare_zone" "delegator" {
+  name = local.domain_to_delegate_from
+}
+
+resource "cloudflare_record" "delegation_ns_record_first" {
```

Comment:
Is there a reason to `not for_each` these records?

Reply:
@fernandoataoldotcom ^^

Reply:
rough - that's a good enough reason to leave as is...